src/main/java/core/basesyntax/strategy/impl/BalanceStorageService.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.StorageService;
+
+public class BalanceStorageService implements StorageService {
```

Comment:
```suggestion
public class BalanceStorageServiceImpl implements StorageService {
```
same for other storageservice implementations